### ISSUE TITLE
JOSS Review: Run Experiments

### ISSUE THREAD

**Issue by GregaVrbancic** [silvandeleemput/memcnn#24]
After installation of memcnn into the fresh python environment, I have followed the [documentation](https://memcnn.readthedocs.io/en/latest/usage.html#run-pytorch-experiments) on how to run the experiments. For me, it was a bit unclear what was the problem with the config.json file, when the first time running the experiment  I experienced the error of config.json not existing error, as in the instructions there is no mention of it. I have fixed this by copying the config.json.example in the required location.

@silvandeleemput  please take a look at this issue and try to address this with improving the documentation on how to run experiments. After improving the documentation you can close this issue and check back in the JOSS review issue.

**Comment by silvandeleemput** [comment]
@GregaVrbancic Thank you for identifying this unclarity. I assume you ran into this issue after installing the package using `pip install` in your environment, which does not automatically set the configuration file (in contrast to the docker images, which do this automatically). I changed the documentation (`readme.rst`, `installation.rst`) to mention the necessity to set the configuration file manually and how to configure it. As a bonus, I also changed `train.py` to mention the configuration file when it is missing or improperly configured, to help the user resolve the issue faster.

Do these changes suffice for closing this issue?
